src/node_v8.cc

```diff
@@ -60,6 +60,14 @@ void GetHeapStatistics(const FunctionCallbackInfo<Value>& args) {
 
 
 void SetFlagsFromString(const FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+  Environment* env = Environment::GetCurrent(isolate);
```

Comment:
You can use the slightly more efficient `Environment::GetCurrent(args)` method here.